assembly/pom.xml

```diff
@@ -174,63 +174,19 @@
                 <descriptor>
                   mvn:org.apache.jclouds.karaf/jclouds-karaf/${jclouds.karaf.version}/xml/features
                 </descriptor>
+                <descriptor>
+                  mvn:org.apache.jclouds.karaf/jclouds-karaf-labs/${jclouds.karaf.version}/xml/features
+                </descriptor>
               </descriptors>
               <features>
                 <feature>config</feature>
-                <feature>jclouds</feature>
-                <feature>jclouds-commands</feature>
-                <feature>jclouds-compute</feature>
-                <feature>jclouds-guice</feature>
```

Comment:
`jclouds-guice`, `jclouds` is included by `jclouds-compute` and it is included by all the compute features. `jclouds-commands` is kept further below. It's like maven transitive features, don't need to mention them if we don't directly depend on them.